docs/cpp-runnable.md

```diff
@@ -4,7 +4,18 @@ id: cpp-runnable
 ---
 
 The `fuse` CLI provides the `run` subcommand for generating executables that
-can be used to test Fuse code without going throught a synthesis workflow.
+can be used to test Fuse code without going through a synthesis workflow.
+
+## Limitation
+
+The backend is meant to be used for functional testing and as such has a few
+important limitations.
+
+1. Exact bitwidth integer computation is not supported. Sized integers get
+   compiled to C++ `int`.
+2. `unroll`s do not create any form of parallelism. The annotation is compiled
+   away after typechecking.
+3. Functions are not inlined.
```

Comment:
These two limitations, unlike the first one, should not affect semantics (only performance).

Reply:
Added clarification.